app.json

```diff
@@ -4,9 +4,9 @@
     "requiredApiVersion": "^1.38.0",
     "iconFile": "icon.png",
     "author": {
-        "name": "Rocket.chat",
-        "homepage": "https://www.rocket.chat/",
-        "support": "support@rocket.chat"
+        "name": "Yuqing Bian",
```

Comment:
Are we sure to change the app author name, homepage and support here?

Reply:
ops, looks like someone  forgot to update the main before :(